docs/docs/platform/handlebars.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 14
+sidebar_label: Handlebars
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Handlebars
+
+You can customize the notification the end users get using Handlebars. This is crucial for many cases, for example, when using the [digest engine](https://docs.novu.co/platform/digest/). 
```

Comment:
=> You can customize the notification (appearance and structure) the end users receive...